### DISCUSSION TITLE
Remixed DX-56 Case for the MiniDexed if you are using the OLED 3306 screen, plus greatly improved DAC placement!

### DISCUSSION THREAD

**Discussion by iluvsa** [probonopd/MiniDexed#701] [General]
Hi All,
   Well, I wanted to contribute to the project, and re-designed the alread-excellent case to accept the OLED 3305 option (these are so cheap now and even if smaller, my eyes seems to do better with them! I also didn't love the floating DAC board if you decided to add that, with the jack just hanging out that back square all willy-nilly, so closed that square up and designed a little parcel shelf for the original case that allow the jack insert into the side of the case and just press fit in! The OLED should just snap in, but depending on the tolerance of your printer, or variance in OLED size, you may have to cut or tape it down, as needed! Case files (Top, Bottom, and Button) are attached in ZIP in the comments!. I can also put it up on Thingiverse or Printables, is the original case available on one of these sites? Then I can just add it as a REMIX.

Oh, and this is the DX-56 case because how are we not calling it that?! I did the Math!

![IMG_5514](https://github.com/user-attachments/assets/426f1b76-2aa3-442c-b466-86f3548d5e72)

**Comment by iluvsa** [comment]
[DX-56.zip](https://github.com/user-attachments/files/16631244/DX-56.zip)

**Reply by blixabloxa** [reply]
Thanks for sharing. It looks great!

**Reply by Merrr** [reply]
Thanks for sharing. I’ve got myself a two tone version.
![IMG_4194](https://github.com/user-attachments/assets/8b3a7513-e3de-4495-8f9a-a46927b16b09)

**Reply by iluvsa** [reply]
This looks great!

**Comment by probonopd** [comment]
> Oh, and this is the DX-56 case because how are we not calling it that?! I did the Math!

Made my day :+1:

**Comment by iluvsa** [comment]
Sometimes you just have to see a joke all the way through, so for those of you with a good 3D printer (this was @ 0.1 mm resolution), here is a DX-56 case with a proper logo! You can multi-color print this, or leave the fill part out and at least have the logo engraved in the plastic - either look pretty good!
![IMG_5529](https://github.com/user-attachments/assets/d8be0c18-8de5-4b8c-8b85-e21e3e0ee3cd)
[DX-56wLogo.zip](https://github.com/user-attachments/files/16656840/DX-56wLogo.zip)

**Comment by pik33** [comment]
I printed this and I cannot close the case - maybe I will try to remix it to use screws instead of latches

Edit: I managed to close the case. It simply needed more force than I wanted to use...

**Reply by iluvsa** [reply]
Oh nice! Yeah, especially with it stuffed with wires it can be a tight squeeze - I found PLA worked a little better because there was a little more flex as compared to PETG - upload pics it's alway great to see them in the wild!

Here's another with a remixed knob:
![IMG_5548](https://github.com/user-attachments/assets/91f98b1a-e318-49bb-9d13-c52a355c0f2a)

**Reply by pik33** [reply]
I have still to remix the case: my OLED display is too long (about 1 mm - width is ok) The knob.. I will test the old design knob using light gold silk filament

**Reply by iluvsa** [reply]
> I have still to remix the case: my OLED display is too long (about 1 mm - width is ok) The knob.. I will test the old design knob using light gold silk filament

Oh let me know if you want me to extend it a little for you - alternatively you could just cut the end of the frame off and slide the OLED through it shoukd still hold fine maybe with a little tape?

**Comment by iluvsa** [comment]
I wonder if there is a little market for these pre-built DX-56's - maybe just to cover costs and the time to build. For those folks that don't want to invest in a printer or a bunch of parts - plus you could get the case in any color you wanted? Let me know how people would feel about that I am not looking to retire on the effort just would be nice to get them into the hands of musicians that just want to PLAY!

**Comment by probonopd** [comment]
How is the DAC board held in, are you using some double-sided tape? Have you tried to make it snap-fit mounted, too?

**Reply by iluvsa** [reply]
@probonopd it’s a press fit now - the DAC output plug slides into the side of the case, and then the whole board kinda snaps into a small square frame - conveniently, when you try and plug a cable into the plug, it presses the board against the frame so it can’t really go anywhere! Hard to explain but I have built 5 of these now and they just kinda “fit” in there!

**Comment by iluvsa** [comment]
And I do mean seeing a joke all the way through - stay tuned, but I am working on a full DX-56 synth that parasitizes an Arturia MiniLab 3 controller, and incorporates both screens without looking like a hack job. Imagine, 8 knobs to control the voices, and sliders for Filter, Resonance, Compression, and Reverb control! Stay tuned!

![IMG_5861](https://github.com/user-attachments/assets/a7bab4d5-342e-4a25-9468-9838b2684b93)

**Reply by soyersoyer** [reply]
This looks cool! Are you going to build an rpi into the MiniLab 3? Have you seen this https://github.com/probonopd/MiniDexed/pull/743 ?

**Reply by iluvsa** [reply]
Yeah, there is a surprising amount of room in there for the Pi, I am deciding whether to have access to the micro sd card, or just stock it full and leave it be inside. There will be access to the audio out jack as well as USB-C ports, in case you wanted to use the controller for another purpose! And next is panel overlay/design, which I will try and make an homage to the original DX-7....
